views/modelIndex/grid.jade

```diff
@@ -0,0 +1,25 @@
+.body
+    div(class="table-responsive")
+        table.table.table-hover
+                thead
+                    tr
+                        th #
+                        if model.list.groupActions && model.list.groupActions.length
+                            th
+                                input(type="checkbox", data-linz-control="checked-all")
+                        if permissions.canDelete !== false || permissions.canEdit !== false || model.list.recordActions.length
+                            th.actions Actions
+                        for field in model.list.fields
+                            th= field.label
+                tbody
+                    for record, index in records
+                        tr
+                            td= from + index + 1
+                            if model.list.groupActions && model.list.groupActions.length
+                                td
+                                    input(type="checkbox", name="ids[]", value="#{record._id}", data-linz-control="checked-record")
+                            if permissions.canDelete !== false || permissions.canEdit !== false || model.list.recordActions.length
```

Comment:
This logic doesn't seem right. I think it should just be `if model.list.recordActions.length`. For a couple of reasons, there might be some custom `recordActions` but they'll never be rendered if `permissions.canDelete === false`.

Also, we're already dealing with `permissions.canDelete` and `permissions.canEdit` within `action-record.jade`...